app/views/shared/_user_fields.html.slim

```diff
@@ -0,0 +1,15 @@
+= simple_form_for @user do |f|
+  - required_new = (action_name == "new")
+  = f.error_notification
+  .form-inputs
+    = f.input :name, required: required_new, autofocus: true, placeholder: "<real name>", hint: "Your actual name"
+    = f.input :tone_name, required: required_new, placeholder: "<stage name>", hint: "What you want to be called"
+    = f.input :email, required: required_new, hint: "also your user id"
+    = f.input :band_start_date, required: false, placeholder: "Date of first performance>", hint: "What was the date of your first performance?",
+      start_year: 2010, end_year: Date.today.year + 1, selected: Date.today.year
+    = f.input :password, required: required_new, autocomplete: "off", label: "New Password", hint: "leave it blank if you don't want to change it"
+    = f.input :password_confirmation, required: required_new, label: "Confirm New Password"
+    = f.input :admin
+  .form-actions
+    = f.button :submit, yield(:button_name)
+    = link_to "Cancel", :back
```

Comment:
you could have this go back to the index action if new, show path if `persisted?` returns true